Noura_Alhadlaq/week_01/day_02/main.js

```diff
@@ -0,0 +1,120 @@
+//what number's bigger?
+var numOne = 1;
+var numTwo = 10;
+
+if(numOne > numTwo){
+    console.log(numOne);
+}else{
+	console.log(numTwo);
+}
+  
+
+//Driving Age 
+
+var age = 12;
+
+if(age < 18){
+    console.log("Sorry, you can't drive yet");
+	var remainigYears = 18 - age;
+	console.log("Sorry, you have " + remainigYears + " years to wait until you can drive");
+}
+else if(age >= 18){
+	console.log("Drive away!");
+}
+
+
+//The World Translator
+
+var lang = "arabic";
+
+if (lang === "english"){
+	console.log("Hello World");
+}else if (lang === "arabic"){
+	console.log("مرحبا بالعالم");
+}else if(lang === "french"){
+	console.log("l'amour");
+}
+
+
+// Suburbs
+
+var testResult = 90;
+
+if( testResult >= 90){
+	console.log("A");
+}else if( testResult >= 80){
+	console.log("B");
+}else if( testResult >= 70){
+	console.log("C");
+}else if( testResult >= 60){
+	console.log("D");
+}else {
+	console.log("F");
+}
+
+
+// Air Conditioning
+
+var currentTemp = 40;
+var acFunctional = false;
+var desiredTemp = 30;
+
+if(acFunctional && currentTemp > desiredTemp){
+	console.log("Turn on the A/C Please");
+} else if(! acFunctional){
+	if(currentTemp > desiredTemp){
+	console.log("Fix the A/C now! It's hot!");
+} else if(currentTemp < desiredTemp){
```

Comment:
Good work here! Something to think about; what will be printed if the `AC is functional` AND `the current temp is less than the desired temp`